README.md

```diff
@@ -28,6 +33,17 @@ Gets a `conanfile.txt` as input and outputs its equivalent `conanfile.py`
 conan migrate:convert-txt conanfile.txt > conanfile.py
 ```
 
+
+### Deployers
+
+#### [Deploy sources](extensions/deployers/sources_deploy.py)
```

Comment:
I would suggest a two line command example, one line the custom command and the second a `ls` without the output, only to indicate where they were installed.